public/friendlycode/js/fc/bramble-ui-bridge.js

```diff
@@ -184,6 +184,49 @@ define(["jquery"], function($) {
       }
     }
 
+    //Publish button
+    $("#navbar-publish-button").click(function() {
+      displayPublishDialog(true);
+    });
+
+    $("#publish-button-cancel").click(function() {
+      displayPublishDialog(false);
+    });
+    $("#publish-underlay").click(function() {
+      displayPublishDialog(false);
+    });
+
+    function displayPublishDialog(visible) {
```

Comment:
Split this out into two functions: `showPublishDialog()` and `hidePublishDialog()`